routes/index.js

```diff
@@ -833,6 +834,155 @@ router.get('/api/v1/equipment', (req, res, next) => {
   });
 });
 
+/*---------------------------- Floor Expenses Endpoints ------------------------------*/
+
+/* GET all floor expenses */
+router.get('/api/v1/floorExpenses', (req, res, next) => {
+  const results = [];
+
+  pg.connect(connectionString, (err, client, done) => {
+    if(err) {
+      done();
+      console;
+      console.log(err);
+      return res.status(500).json({success: false, data: "You did something so bad you broke the server =("});
+    }
+
+    const query = client.query('SELECT * FROM floorExpenses ORDER BY floor_expense_id ASC;');
+    
+    query.on('row', (row) => {
+      results.push(row);
+    });
+
+    query.on('end', () => {
+      done();
+      return res.json(results);
+    });
+  });
+});
+/* POST a new floor expense */
+router.post('/api/v1/floorExpense', urlencodedParser, function(req, res, next) {
+  const results= [];
+
+  pg.connect(connectionString, (err, client, done) => {
+
+    if(err) {
+      done();
+      console.log(err);
+      return res.status(500).json({success: false, data: err, body: req.body});
+    }
+
+
+    var reqJson = req.body;
+    console.log(reqJson);
+    var firstQuery = createNewEntryQuery(reqJson, 'floorExpenses');
+
+    var colValues = [];
+    Object.keys(reqJson).filter(function (key) {
+      colValues.push(reqJson[key]);
+    });
+
+    console.log(firstQuery);
+
+    client.query(firstQuery, colValues);
+
+    const query = client.query('SELECT * FROM floorExpenses, floorMoney WHERE floorExpenses.event_description = $1 and floorExpenses.amount = $2 and floorExpenses.turned_in_date = $3 and floorExpenses.processed_date = $4 and floorMoney.hall_and_floor = $5 and floorMoney.floorMoney_id = floorExpenses.floor_id', [reqJson.event_description, reqJson.amount, reqJson.turned_in_date, reqJson.processed_date, reqJson.hall_and_floor] )
```

Comment:
I don't think the query is returning the row, so a response doesn't have the newly created row in its body.